cetmix_tower_server/views/cx_tower_server_template_view.xml

```diff
@@ -56,7 +57,10 @@
                             <div class="oe_kanban_global_click">
                                 <div class="col-12 pt8 o_kanban_primary_right">
                                     <div>
-                                        <field name="reference" />
+                                        <strong>Servers:</strong>
```

Comment:
Show server count only if number of servers  > 0.
NB: use "t-if" instead of "attrs invisible"

Reply:
Please also replace "attris invisible" with "t-if" for all other conditionally shown elements in the kanban view.
Same as it is done for the [Server](https://github.com/cetmix/cetmix-tower/blob/9691c9acfce5d284f99dff70a36f9016d3a52d6b/cetmix_tower_server/views/cx_tower_server_view.xml#L31C20-L31C21)